.github/actions/integration-tests/run-ci-stage1

```diff
@@ -513,6 +532,10 @@ run_cmd "crucible es init"
 run_cmd "crucible es rebuild"
 
 for scenario in $(echo "${CI_SCENARIOS}" | sed -e "s/,/ /g"); do
+    if [ "${scenario}" == "multi" ]; then
+        scenario="iperf-and-uperf"
```

Comment:
what?? is this going to work?

Reply:
Yes, it does work :)

In this loop the `${scenario}` variable is being used to match the directory/file names that are created for a run result.  The `multi` scenario is a special case in that its an alias for a job with multiple benchmarks running, in this case iperf and uperf.  When Crucible does a multiple benchmark run it creates the result directory name in the form `<benchmark-1>-and-<benchmark-2>` so the `multi` alias has to be resolved to that in a special case since it does not conform to the same behavior that all the other scenarios do.